e2e/testdata/root-sync-helm-chart-cr.yaml

```diff
@@ -42,6 +42,10 @@ spec:
         limits:
           cpu: 1
           memory: 300Mi
+      mariadb:
+        primary:
+          persistence:
+            enabled: false
```

Comment:
Nice! Will we still need to manually clean up the lingering `PVC` resources on the clusters from previous runs?

Reply:
The cleanup function is no longer needed and removed.

Reply:
I mean specifically for the resources that were created before this change, i.e. test runs on commits before this change is submitted. Will we need to manually clean up those?

Reply:
That should not be needed as the mariadb doesn't check the persisted data, but it is good to delete the test data anyway.
I'll remove them later once the first CI job passes after this PR is merged.